src/mobile/src/components/Banner/BannerText.jsx

```diff
@@ -25,20 +25,41 @@ const styles = StyleSheet.create({
   },
 });
 
+async function getStudentQuotes() {
+  const { data, error } = await supabase.from('quotes').select('*');
+
+  if (error) {
+    throw new Error(error.message);
+  }
+
+  return data;
+}
+
 const BannerText = () => {
-  const [studentQuote, setStudentQuote] = useState(quotes[0]);
+  const [studentQuotes, setStudentQuotes] = useState({});
```

Comment:
To know if a quote has been set, you want to initialize this to a nullish value and then conditionally render the quote on line 54

Reply:
```suggestion
  const [studentQuotes, setStudentQuotes] = useState({
  blog_url: '', 
  quote: '',
  ....
  });
```
I this what you mean ? @DukeManh